Cython/Debugger/libcython.py

```diff
@@ -231,16 +231,17 @@ def get_cython_lineno(self, frame):
         correspondence between the C and Cython code.
         """
         cyfunc = self.get_cython_function(frame)
-        return cyfunc.module.lineno_c2cy.get(self.get_c_lineno(frame), 0)
+        return cyfunc.module.lineno_c2cy.get(self.get_c_lineno(frame), (None, 0))
```

Comment:
`lineno_c2cy` is a mapping to a tuple `(src_path, src_lineno)`, returning just `0` was a legacy of when it was just a mapping of line numbers